README.rst

```diff
@@ -58,6 +58,7 @@ Key Features
     * `Question answering <https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/nlp/question_answering.html>`_
     * `GLUE benchmark <https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/nlp/glue_benchmark.html>`_
     * `Information retrieval <https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/nlp/information_retrieval.html>`_
+    * `Entity Linking <https://github.com/NVIDIA/NeMo/blob/entity-linking-documentation/docs/source/nlp/entity_linking.rst>`_
```

Comment:
Need to change this to link to a page at docs.nvidia.com/deeplearning/nemo instead of directly to the .rst file in git. How do I do this?

Reply:
I think you can anticipate what URL this will map to for example `Information retrieval <https://docs.nvidia.com/deeplearning/nemo/user-guide/docs/en/main/nlp/entity_linking.html`? @okuchaiev can confirm maybe?

Reply:
yes, feel free to put link to docs on main here